### PR TITLE
[UX] Minor UI tweaks

### PR DESCRIPTION
**Pull request by niels9001**
## Description
Typography:
- Make the copy button always visible
- Optimized fontsizes for better readability

Colors:
- Small fixes to color labels/brush names
- Replace the default copy button with the `CopyButton`.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml [hunk 1]
```diff
@@ -39,8 +39,8 @@
         </Grid.RowDefinitions>
         <RichTextBlock>
             <Paragraph>
-                Type helps provide structure and hierarchy to UI. The default font for Windows is
-                <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>.
+                Type helps provide structure and hierarchy to UI. The default font for Windows is <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>
+                .
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml:43]: The line break before the period will be converted to a space . Visual Studio keeps doing this on every "format document" . :⁠-⁠(

File: WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml [hunk 2]
```diff
@@ -170,7 +170,7 @@
                         Grid.Column="2"
                         Foreground="{ThemeResource TextFillColorSecondaryBrush}"
                         Style="{StaticResource CaptionTextBlockStyle}"
-                        Text="Size / Line height" />
+                        Text="Size" />
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml:173]: Why remove the text "line height "? Now there will be two numbers who both mean font size at the same time??

**Review by chingucoding — COMMENTED** [review]
I can only second what @Jay-o-Way said, otherwise this looks fine.

**Comment by niels9001** [comment]
@chingucoding @jay-o-way Thanks! Addressed your feedback!

**Comment by karkarl** [comment]
Please rebase from main to get the latest CI build fixes and run an Accessibility Insights pass on the updated xaml pages. (Automated test pass coming soon!)

**Comment by Jay-o-Way** [comment]
What's keeping this?

**Review by karkarl — APPROVED** [review]

File: WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml (hunk 1 above)
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml:43]:
```suggestion
                Type helps provide structure and hierarchy to UI. The default font for Windows is <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>.
```

**Comment by karkarl** [comment]
@niels9001 Can you rebase from main to get the test failure fix?

**Comment by niels9001** [comment]
> @niels9001 Can you rebase from main to get the test failure fix?

Done! CI should hopefully pass now :)

**Review by chingucoding — APPROVED** [review]

**Pull request merged**